### ISSUE TITLE
[a11y] 9.1.4.10 Break content

### ISSUE THREAD

**Issue by tschiebel** [owncloud/web#10728]
audit: https://infinite.owncloud.com/s/PnaAukHGGsLnrMM

https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-1-4-10-inhalte-brechen-um

### Personal 

- [ ] With a browser window width of 1280 CSS pixels and a zoom magnification of
400%, the page content does not break. This means that all information and
functions are not available without users having to scroll horizontally.
![image](https://github.com/owncloud/web/assets/132444655/9b38ef20-1531-4bf8-9f67-bc6cb751f622)

### Shares 

- [ ] If the browser window width is set to 1280 CSS pixels and the zoom magnification is set to
400%, only two columns are displayed in the table: "Name" and "Actions".
"Actions". Other table columns are no longer accessible to users. To
ensure that all table columns are visible even at higher zoom magnification, the table should either be
the table should either be designed in such a way that it adapts to the available
width or offer the option of scrolling horizontally through the table columns.
scroll horizontally through the table columns.
![image](https://github.com/owncloud/web/assets/132444655/49246a1c-7919-478c-80f1-df46ae213a7a)

### Account 

- [x] If the browser window width is set to 1280 CSS pixels and the zoom magnification is set to
400%, the content at the bottom of the page is cut off and is no longer accessible.
no longer accessible.
![image](https://github.com/owncloud/web/assets/132444655/8ca38acd-70db-4529-be1e-ff182b353b36)

**Comment by pascalwengerter** [comment]
Assigning @tbsbdr here since I need a decision on how to handle the tables on small screens

**Comment by tbsbdr** [comment]
- remove text from buttons "New" and "Upload" for like 1280@400% 

![Image](https://github.com/user-attachments/assets/c868f7e0-f425-4d64-87ab-811c579f6de7)


- **Shared** and **Spaces** should not scroll horizontally (lets make it like in personal, i.e. stack and remove)


![Image](https://github.com/user-attachments/assets/aece5ae5-9928-41ec-bddc-2b9a89ce0b5a)


![Image](https://github.com/user-attachments/assets/f674ecc3-198a-499d-b285-4dade2708db1)

**Comment by kulmann** [comment]
@tbsbdr at 400% zoom on a 14 inch macbook the file list is not visible at all 🙈 see screenshot:

<img width="1624" alt="Screenshot 2024-08-05 at 05 42 55" src="https://github.com/user-attachments/assets/ba0264a2-96db-4222-b9ff-df38fc785c6a">

The table header is visible. Since the area above the table doesn't scroll I can't scroll the actual files into view. At 300% it still kind of works - I can see two rows of the file list (see next screenshot):

<img width="1624" alt="Screenshot 2024-08-05 at 05 46 13" src="https://github.com/user-attachments/assets/665017a9-8c27-45f7-8b3c-25582e250bd7">

Do we have to fix this or can we assume a different browser window height if someone uses 400% zoom? 🤔

**Comment by tbsbdr** [comment]
uff, how about making the files top bar scrollable eg. if a certain zoom is reached? just thinking aloud..

**Comment by JammingBen** [comment]
Or just "unstick" the area above the table?

**Comment by tbsbdr** [comment]
eh, yes, thats what I wanted to say :)

**Comment by kulmann** [comment]
Yep, good idea. For normal desktop resolutions and 100% zoom level I like the sticky header. But for other situations that would be probably the easiest and most accessible solution.

**Comment by JammingBen** [comment]
> Yep, good idea. For normal desktop resolutions and 100% zoom level I like the sticky header. But for other situations that would be probably the easiest and most accessible solution.

I'm just not sure if we should couple this to the zoom level. Some users have a default zoom setting of e.g. 110% or 125%. I feel like we still want to have the area sticky in this case.

Maybe couple it to the screen width? Because that's what basically happens with a zoom level of 400%: the screen width gets extremely small. So I suggest only doing this for screen widths <420px or <640px.
